.eslintrc.json

```diff
@@ -1,11 +1,13 @@
 {
   "root": true,
+  "plugins": ["ssr-friendly"],
```

Comment:
Should we include the new plugin in js/src only?

Reply:
oh yeah it should be better but didn't found it when I looked for 😅